suite-common/wallet-core/src/fees/feesReducer.ts

```diff
@@ -40,3 +41,33 @@ export const feesReducer = createReducer(initialState, builder => {
 
 export const selectNetworkFeeInfo = (state: FeesRootState, networkSymbol?: NetworkSymbol) =>
     networkSymbol ? state.wallet.fees[networkSymbol] : null;
+
+export const selectNetworkFeeLevelTimeEstimate = (
+    state: FeesRootState,
+    level: FeeLevelLabel,
+    networkSymbol?: NetworkSymbol,
+): string | null => {
+    const networkFeeInfo = selectNetworkFeeInfo(state, networkSymbol);
+    if (!networkFeeInfo) return null;
+
+    const feeLevel = networkFeeInfo.levels.find(x => x.label === level);
+
+    if (!feeLevel) return null;
+
+    return formatDuration(networkFeeInfo.blockTime * feeLevel.blocks * 60);
+};
+
+export const selectNetworkFeeLevelFeePerUnit = (
```

Comment:
The code in `selectNetworkFeeLevelTimeEstimate` and `selectNetworkFeeLevelFeePerUnit` looks very similar. It looks like a candidate for separate selector, that can be reused. Wdyt?

Reply:
makes sense 👍 3a2264530a